src/commands/apps-sdkconfig.ts

```diff
@@ -0,0 +1,117 @@
+import * as ora from "ora";
+import * as fs from "fs-extra";
+
+import * as Command from "../command";
+import {
+  AppMetadata,
+  AppPlatform,
+  getAppConfig,
+  getAppPlatform,
+  listFirebaseApps,
+} from "../management/apps";
+import * as FirebaseError from "../error";
+import * as requireAuth from "../requireAuth";
+import * as logger from "../logger";
+import { promptOnce } from "../prompt";
+
+async function selectAppInteractively(
+  options: any,
+  appPlatform: AppPlatform
+): Promise<AppMetadata> {
+  let projectId = options.project;
+  if (!projectId) {
+    projectId = await promptOnce({
```

Comment:
isn't there a function that exists now to select your project?

to that end, why not make `projectId` a required parameter to this function? this is now doing _two_ things (selecting a project and selecting an app) when it advertised one in its name "selectAppInteractively"

Reply:
1. https://github.com/firebase/firebase-tools/pull/1506 is still review, so I would put a TODO here to replace this code. @mbleigh, @nbegley, Do you think it's good here to call another API method listing projects? 
2. Moving `projectId` prompt outside of this function

Reply:
Your TODO LGTM, and I think it covers what Bryan was asking for.